src/qibolab/platform/platform.py

```diff
@@ -21,7 +20,7 @@
 
 InstrumentMap = dict[InstrumentId, Instrument]
 QubitMap = dict[QubitId, Qubit]
-CouplerMap = dict[QubitId, Coupler]
+CouplerMap = dict[QubitId, Qubit]
```

Comment:
Isn't `QubitMap` now sufficient? Maybe for the time being it is easier to leave as it is, since `qubits` and `couplers` are still separate in the `Platform`. Given that there is an open discussion about merging them, we could leave the removal of `CouplerMap` for when this is done.

Reply:
Yes, you're right that `QubitMap` should be sufficient.

Essentially, in this PR I haven't been so much thoughtful about further couplers handling optimizations, and just aimed to remove the separate class on its own.

Types that look the same could be conceptually different (we could have a `QubitId` being just an `int`, but then an `int` is a `QubitId` only in certain situations), but in this case they are really close to each other. So, yes, they could be the same.

In any case, since we have the open discussion about how to treat couplers in general, let's keep it for there.